packages/mwp-config/stylelint/stylelint-config-swarmProject.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+	rules: {
```

Comment:
I had that at first, but thought it might be better to explicitly extend `stylelint-config-base` _and_ `stylelint-config-swarmProject` in SDS projects. Happy to update though :)